File: 프로그래머스/lv1/12944. 평균 구하기/평균 구하기.py
def solution(arr):
    # 분모가 0인 경우가 존재하면 안되기 때문에 조건 추가
    # 단, 제한사항에 arr의 길이가 최소 1이기 때문에 사실상 필요 없음
    if len(arr) == 0:
        return 0
    return sum(arr) / len(arr)

File: 프로그래머스/1/12944. 평균 구하기/평균 구하기.java
class Solution {
    public double solution(int[] arr) {
        return getArraySum(arr);
    }
    
    public double getArraySum(Object obj) {
        int amount = 0;
        int[] arr = (int[])obj;
        
        for(int number:arr)
            amount += number;

        return amount/Double.valueOf(arr.length);
    }
}